main.go

```diff
@@ -241,7 +255,12 @@ func initServer(logger *zap.Logger, apmTracer *apm.Tracer, config *Config) *http
 	router := mustLoadRouter(logger, config, indexer)
 	apmgorilla.Instrument(router, apmgorilla.WithTracer(apmTracer))
 
-	return &http.Server{Addr: address, Handler: router}
+	if tlsMinVersion == "" {
+		return &http.Server{Addr: address, Handler: router}
+	}
+
+	TLSConfig := &tls.Config{MinVersion: tlsMinVersionCode}
+	return &http.Server{Addr: address, Handler: router, TLSConfig: TLSConfig}
```

Comment:
Nit. Lower case name of local variable.
```suggestion
	tlsConfig := &tls.Config{MinVersion: tlsMinVersionCode}
	return &http.Server{Addr: address, Handler: router, TLSConfig: tlsConfig}
```